### ISSUE TITLE
Like operator in the 'where' expression doesn't work

### ISSUE THREAD

**Issue by surik** [xerions/ecdo#10]
Because

``` elixir
Code.string_to_quoted("a like 2")
{:ok, {:a, [line: 1], [{:like, [line: 1], [2]}]}}
```

doesn't match to `Ecdo.Builder.Where:to_ecto_ast/3`.
I think there are same problems with `:not`, `:or`, `:and`, and `:ilike`

**Comment by surik** [comment]
Ok, this is my mistake. Now we use `like` as function: `like(a, "%2%")`
But there is problem with: `like(a, "%2%") or like(a, "%3%")`.
I continue my investigation.

**Comment by liveforeverx** [comment]
Fixed in 9399246afa64f188807e54b0e7675be0e67310d8
